woocommerce/pinterest-for-woocommerce
# Detect other tags installed

### Changes proposed in this Pull Request:
Closes #417.

There are other tags that may potentially cause conflict with the plugin. We are showing warning messages, so the customer can decide if removing the other tags or disable the plugin's tracking.

The current PR adds detection for the following tags:
- Google Tag Manager.
- PixelYourSite - Pinterest Addon.

As mentioned by @jconroy [here](https://github.com/woocommerce/pinterest-for-woocommerce/issues/417#issuecomment-1173745843), in order to keep this PR small we can add support for more third party tags in other PRs.

### Screenshots:

![pinterest-warning-catalog](https://user-images.githubusercontent.com/40774170/178590006-4156b65e-e5d5-4ff0-8301-6719133c3c20.png)
![pinterest-warning-tracking](https://user-images.githubusercontent.com/40774170/178590011-b391ec30-2260-461b-8fb4-850b75359a81.png)

### Detailed test instructions:
1. Install the plugin and do the onboarding.
2. Install and enable either Google Tag Manager or PixelYourSite - Pinterest Addon plugins.
3. The warning messages should be displayed on the Catalog tab and the Tracking section (Connection tab).
4. After disabling/uninstalling the third party extensions the warning should not be displayed.

### Additional details:


### Changelog entry

> Add - Warning message on potential conflicting tags.
